### ISSUE TITLE
Components are rendered with unnatural width due to display flex

### ISSUE THREAD

**Issue by ArthurClemens** [phenixdigital/phx_live_storybook#178]
This alert component is a regular div and should expand to the full width of its container. However, in the rendered story the parent element is styled with `display: flex` which results in an unnatural small size: 

![image](https://user-images.githubusercontent.com/350303/201532551-e1e56b98-cf53-4b3e-8257-3515f98d3b46.png)

**Comment by cblavier** [comment]
Hey Arthur, does the new `container` option does the trick for you?

https://github.com/phenixdigital/phx_live_storybook/blob/main/guides/components.md#container

This is not released yet, only on `main` branch

**Comment by ArthurClemens** [comment]
I can't get the main branch to run.

- Changed `deps` to `{:phx_live_storybook, git: "https://github.com/phenixdigital/phx_live_storybook", branch: "main"}`
- Removed `build` and `deps` folders
- Called `mix deps.get`

Did something change in the setup?

**Comment by cblavier** [comment]
Any error? 
Did you follow instructions in https://github.com/phenixdigital/phx_live_storybook#contributing ?

**Comment by ArthurClemens** [comment]
You are right, I should do that first (instead of making quick changes during my working day). Will follow up later.

**Comment by ArthurClemens** [comment]
Yes, `container` does solve my problem. Still, looks like a good default to me...
